packages/babel-preset-react-app/create.js

```diff
@@ -177,24 +142,11 @@ module.exports = function(api, opts, env) {
       ],
       // Adds syntax support for import()
       require('@babel/plugin-syntax-dynamic-import').default,
-      isEnvTest &&
-        // Transform dynamic import to require
-        require('babel-plugin-dynamic-import-node'),
-    ].filter(Boolean),
-    overrides: [
-      isFlowEnabled && {
-        exclude: /\.tsx?$/,
-        plugins: [require('@babel/plugin-transform-flow-strip-types').default],
-      },
-      isTypeScriptEnabled && {
-        test: /\.tsx?$/,
-        plugins: [
-          [
-            require('@babel/plugin-proposal-decorators').default,
-            { legacy: true },
-          ],
-        ],
-      },
+      // Transform dynamic import to require
+      isEnvTest && require('babel-plugin-dynamic-import-node'),
+      require('@babel/plugin-transform-react-display-name'),
+      isEnvDevelopment && require('@babel/plugin-transform-react-jsx-source'),
+      isEnvDevelopment && require('@babel/plugin-transform-react-jsx-self'),
```

Comment:
are these recommended by emotion, or just ones you think would be helpful?

Reply:
I think they're helpful.

- [plugin-transform-react-display-name](https://babeljs.io/docs/en/babel-plugin-transform-react-display-name) helps in react dev tools and debugging, as it adds the name of the react class to the instance.
- [plugin-transform-react-jsx-source](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx-source) adds some sourcemap helps to be picked-up (I think) by react dev tools.
- Honestly, I'm not exactly sure what [plugin-transform-react-jsx-self](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx-self) is used for, but it seems trivial to leave it there, so I did.